test/System.CommandLine.StaticCompletions.Tests/System.CommandLine.StaticCompletions.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net10.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsTestProject>true</IsTestProject>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- None of these deps should be hard-coded, but something's wonky unless I opt into the whole massive Microsoft.NET.TestFramework mess.
+         I don't want that. This should be able to stand alone. -->
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.24528.1"/>
+    <PackageReference Include="FluentAssertions" Version="6.12.0" />
+    <PackageReference Include="XUnit" Version="2.9.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.13.0-preview-24610-01" />
+    <PackageReference Include="Microsoft.TestPlatform" Version="17.13.0-preview-24610-01" />
+    <PackageReference Include="Verify.Xunit" Version="25.0.2" />
+    <PackageReference Include="Verify.DiffPlex" Version="3.0.0" />
```

Comment:
Note for the SDK team: I'm trying on purpose to keep this test project as slim as possible, but that has two side-effects that I'm having trouble reasoning about:

* the package versions aren't coming through in the way I'd expect, requiring this duplication
* I suspect Helix is mad due to some missing convention

I'm really hoping to not have to take a dependency on the rather large SDK test framework project because that massively expands cycle times.

Reply:
Does central package versioning not get us the versions? Or rather, why not?